fastlane/Fastfile

```diff
@@ -472,6 +479,28 @@ platform :ios do
     }
   end
 
+  desc "Enable Customer Center"
+  lane :enable_customer_center do
+    # This enables the customer center by replacing CUSTOMER_CENTER_ENABLED with true in all files. 
+    # This lane should be removed once Customer Center is released.
+    require 'find'
+
+    project_root = File.expand_path("../", __dir__)
+
+    Find.find(project_root) do |path|
+      # process only Swift files
+      next unless File.file?(path) && File.extname(path) == ".swift"
+
+      content = File.read(path)
+
+      updated_content = content.gsub('CUSTOMER_CENTER_ENABLED', 'true')
+
+      File.write(path, updated_content) if content != updated_content
+    end
```

Comment:
would have been shorter as a find + sed, but this is much easier to read IMO

Reply:
Hmm I guess the fastlane commands to run tests are only run through CI normally, so this is probably ok... We could also make a sort of "block" so we restore it once tests finish, but I don't think it's worth it 👍